app/domain/logs.rb

```diff
@@ -277,65 +277,65 @@ module AuthnJwt
 
       IssuerResourceNameConfiguration = ::Util::TrackableLogMessageClass.new(
         msg: "\"issuer\" value will be taken from '{0-resource-id}'",
-        code: "CONJ00054D"
+        code: "CONJ00054I"
       )
 
       RetrievedIssuerValue = ::Util::TrackableLogMessageClass.new(
         msg: "Retrieved \"issuer\" with value '{0}'",
-        code: "CONJ00055D"
+        code: "CONJ00055I"
       )
 
       ParsingIssuerFromUri = ::Util::TrackableLogMessageClass.new(
         msg: "Parsing \"issuer\" value from '{0}'...",
         code: "CONJ00056D"
       )
 
-      JWTAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
-        msg: "Started JWT authentication flow for vendor '{0}'",
-        code: "CONJ00057D"
+      JwtAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
+        msg: "Started JWT authentication flow for authenticator '{0-authenticator-name}'",
+        code: "CONJ00057I"
       )
 
-      URL_IDENTITY_PROVIDER_SELECTED = ::Util::TrackableLogMessageClass.new(
-        msg: "The JWT ID is included in and will be retrieved from the URL",
+      SelectingIdentityProviderInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Selecting identity provider interface...",
         code: "CONJ00058D"
       )
 
-      DECODED_TOKEN_IDENTITY_PROVIDER_SELECTED = ::Util::TrackableLogMessageClass.new(
-        msg: "JWT Identity in decoded token is available and will be retrieved from it",
-        code: "CONJ00059D"
+      SelectedIdentityProviderInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Selected identity provider interface: '{0-identity-provider-interface-name}'",
+        code: "CONJ00059I"
       )
 
-      LOOKING_FOR_IDENTITY_FIELD_NAME = ::Util::TrackableLogMessageClass.new(
+      LookingForIdentityFieldName = ::Util::TrackableLogMessageClass.new(
         msg: "Looking for variable field name in '{0}'...",
```

Comment:
what does this mean?

Reply:
Correct i do not like this log,
deleted and changed to:
`CONJ00060I Retrieved value 'sub' of resource name 'cucumber:variable:conjur/authn-jwt/okta/token-app-property'
`